### ISSUE TITLE
PWM doesn't work when the v4.11+ kernel is used

### ISSUE THREAD

**Issue by fivdi** [julianduque/beaglebone-io#45]
This issue is related to the location of the files used to control PWM. As an example, the glob pattern for the `duty_cycle` file on v4.4.x and v4.9.x is:
```
/sys/devices/platform/ocp/48302000.epwmss/48302200.pwm/pwm/pwmchip*/pwm0/duty_cycle
```
on v4.14.x it's:
```
/sys/devices/platform/ocp/48302000.epwmss/48302200.pwm/pwm/pwmchip*/pwm-*:0/duty_cycle
```

**Comment by fivdi** [comment]
It looks like this issue is present in v4.11+. See https://github.com/jadonk/bonescript/issues/156 for details.

**Comment by fivdi** [comment]
Fixed by #51.
